tests/ui_tests/app/page/common/configuration/page.py

```diff
@@ -79,37 +91,63 @@ def get_input_value(self, adcm_test_attr_value: str, is_password: bool = False)
 
     @allure.step('Check input of field "{adcm_test_attr_value}" has value "{expected_value}"')
     def assert_input_value_is(
-        self, expected_value: str, adcm_test_attr_value: str, is_password: bool = False
+        self,
+        expected_value: str,
+        adcm_test_attr_value: Optional[str] = None,
+        row: Optional[WebElement] = None,
+        is_password: bool = False,
     ):
         """
         Assert that value in field is expected_value (using retries)
 
         :param expected_value: Value expected to be in input field
         :param adcm_test_attr_value: Value of attribute "adcm_test" to generate Locator
+        :param row: row with required input
         :param is_password: Is field password/confirmation
         """
 
         def assert_value():
-            assert expected_value == self.get_input_value(
-                adcm_test_attr_value, is_password
-            ), f'Value in {adcm_test_attr_value} field should be empty after reset'
+            input_value = (
+                self.get_input_value(adcm_test_attr_value, is_password)
+                if adcm_test_attr_value
+                else self.get_config_row_info(row).value
+            )
+            assert (
+                expected_value == input_value
+            ), f'Expected value was {expected_value} but presented is {input_value}'
 
         wait_until_step_succeeds(assert_value, timeout=4, period=0.5)
 
-    def reset_to_default(self, adcm_test: str):
+    def reset_to_default(self, adcm_test: Optional[str] = None, row: Optional[WebElement] = None):
         """Click reset button"""
-        self.find_and_click(self.config.reset_btn(adcm_test))
+        if adcm_test:
+            self.find_and_click(self.config.reset_btn(adcm_test))
+        elif row:
+            self.find_child(row, CommonConfigMenu.ConfigRow.reset_btn)
+        else:
+            raise AssertionError("Need to indicate adcm_test or row")
```

Comment:
Not `AssertionError`, because it should fail as broken test. Consider adding custom Error or use something like `ValueError`

Reply:
changed. thnx!